src/tilesource.js

```diff
@@ -436,10 +436,11 @@ $.TileSource.prototype = {
      * Responsible for retrieving, and caching the
      * image metadata pertinent to this TileSources implementation.
      * There are three scenarios of opening a tile source:
```

Comment:
This line no longer makes sense as it's not followed by a list of 3 scenarios.

Reply:
Thanks, will fix.